recipe/meta.yaml

```diff
@@ -18,9 +18,9 @@ requirements:
   build:
     - curl           # [win]
     - 7za            # [win]
-    - vc             # [win]
+    - vc{{ VC_VERSION }}   # [win]
```

Comment:
Somewhat confusingly, the feature has no space - it is different from the package (only ```vc``` - but with associated version)